packages/auth-providers/supertokens/setup/src/templates/api/lib/supertokens.ts.template

```diff
@@ -9,10 +9,11 @@ const apiGatewayPath =
   process.env.SUPERTOKENS_API_GATEWAY_PATH || '/.redwood/functions'
 
 export const config: TypeInput = {
+  # The below options are ok here even if you're not running on top of AWS Lambda, since Redwood internally translates Fastify request/response objects to and from the AWS Lambda format.
```

Comment:
I got confused by the `awsLambda`/serverless options below, probably worth mentioning this within the template?

Reply:
If user's aren't using AWS lambda to deploy their app, which other web framework can be used to deploy theie app?

Reply:
Redwood supports serverful deploys on for example Fly.io or Render. There the api side runs ontop of Fastify.

Reply:
Right. In that case, the framework should be fastify since we support that as well. So users would need to select what they would be using.

Reply:
Are you sure? As the text says, RW transforms everything to make it look like it's running in AWS lambdas even when it's running on Fastify. Just so our users don't have to worry about it.

Reply:
Right ok. Then awsLambda would continue to work.